code/web/src/modules/subscription/api/actions.js

```diff
@@ -29,7 +29,7 @@ export function getList(isLoading = true) {
 
     return axios.post(routeApi, query({
       operation: 'subscriptions',
-      fields: ['id', 'user { name, email }', 'crate { id, name, description }', 'createdAt']
+      fields: ['id', 'user { name, email, style }', 'crate { id, name, description }', 'createdAt']
```

Comment:
This, and the following similar lines, incorporate the new style property to the app store.